aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -102,6 +102,18 @@ The `Add-Migration` command creates code to create the initial database schema.
 
 The `Update-Database` command runs the `Up` method in the *Migrations/\<time-stamp>_InitialCreate.cs* file, which creates the database.
 
+You can perform the preceeding steps using the command-line interface (CLI) rather than the PMC:
```

Comment:
This is kind of lost without special formatting like a Note.  Also might consider noting this earlier, or people could get frustrated doing the instructions and not realize there's an alternative.